ansible/configs/ocp-clientvm/post_software.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Step 00xxxxx post software
+  hosts: bastions
+  become: yes
+  gather_facts: False
+  vars_files:
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_vars.yml"
+  tasks:
+    - debug:
+      msg: "Post-Software Steps starting"
+
+- name: PostSoftware flight-check
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  vars_files:
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_vars.yml"
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_secret_vars.yml"
+  tags:
+    - post_flight_check
+  tasks:
+    - debug:
+        msg: "Post-Software checks completed successfully"
+
+- name: Gather facts
+  hosts:
+  - all
+  vars_files:
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_vars.yml"
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_secret_vars.yml"
+  gather_facts: true
+  tags:
+    - ocp_report
+
+- name: Generate reports
+  hosts: localhost
+  connection: local
+  become: false
+
+  vars_files:
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_vars.yml"
+    - "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/env_secret_vars.yml"
+  tags:
+    - ocp_report
+  vars:
+    env_all_hosts: all
+  tasks:
+    - name: get repo version used to deploy
+      command: git rev-parse HEAD
+      args:
+        chdir: "{{ ANSIBLE_REPO_PATH }}"
+      register: ansible_agnostic_deployer_head
+
+    - name: Gather ec2 facts
+      ec2_remote_facts:
+        aws_access_key: "{{ aws_access_key_id }}"
+        aws_secret_key: "{{ aws_secret_access_key }}"
+        region: "{{ aws_region }}"
+      when: ocp_report
+    - name: Generate report
+      template:
+        src: "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/files/ocp_report.adoc.j2"
+        dest: "{{ ANSIBLE_REPO_PATH }}/workdir/ocp_report_{{ env_type }}-{{ guid }}.adoc"
+      when: ocp_report
```

Comment:
do you really need the ocp report for this single-instance config ?

Reply:
Probably not. Don't know what it does - so I can take the whole section out.